services/121-service/src/shared/const.ts

```diff
@@ -0,0 +1,43 @@
+const nameConstraintQuestionsArray = [
```

Comment:
This constant should be used in the test, instead of copy-pasting the values only.